tests/contrib/grpc_aio/test_grpc_aio.py

```diff
@@ -396,6 +396,7 @@ async def test_analytics_without_rate(server_info, tracer):
     assert server_span.get_metric(ANALYTICS_SAMPLE_RATE_KEY) == 1.0
 
 
+@pytest.mark.skipif(sys.version_info == (3, 11, 0), reason="Running this on Python 3.11 results in segmentation faults")
```

Comment:
Add/link upstream issue

Reply:
Document in supported libraries with asterisk (explain why xyz library doesn't support 3.11)